revoke_handler.go

```diff
@@ -125,7 +124,7 @@ func (f *Fosite) WriteRevocationResponse(rw http.ResponseWriter, err error) {
 
 		rw.WriteHeader(ErrInvalidClient.Code)
 		rw.Write(js)
-	default:
+	} else {
 		// 200 OK
 		rw.WriteHeader(http.StatusOK)
```

Comment:
Shouldn't this return an error code ? If I'm here it means that there is an error so we should return it to the client shouldn't we ?

Reply:
So this is not something I changed, but it is like this already in existing code. But you are right, I was also wondering about this. I can fix this as part of this PR as well. What should be returned here? Internal server error?

Reply:
503 from the RFC. But I think they are on it. One of my colleague signal it already.

Reply:
We received exactly this as a sec vulnerability disclosure yesterday. Working on that already, stay tuned.